manifests/init.pp

```diff
@@ -47,6 +48,12 @@
     $my_package_latest = $package_latest
   }
 
+  if is_bool($package_manage) == true {
```

Comment:
@ghoneycutt I asked @ofsole to use this pattern. I prefer it because of better handling of wrong types.